stdlib/3/urllib/response.pyi

```diff
@@ -1,12 +1,14 @@
 # private module, we only expose what's needed
 
-from typing import BinaryIO, Iterable, List, Mapping, Optional, Type, TypeVar
+from typing import Any, BinaryIO, Callable, Iterable, List, Mapping, Optional, Tuple, Type, TypeVar
 from email.message import Message
 from types import TracebackType
 
 _AIUT = TypeVar("_AIUT", bound=addbase)
 
 class addbase(BinaryIO):
+    fp: BinaryIO
```

Comment:
Should probably be `IO[bytes]` instead which is slightly wider.

Reply:
Should all instances of BinaryIO be IO[bytes] instead?

Reply:
I want to say yes but presumably there's a reason `BinaryIO` exists. At least in this file I don't see why we can't go with `IO[bytes]`.

Reply:
Actually, I think I need to push back on this. `BinaryIO` has a small-but-significant change in the type signature for [write()](https://github.com/python/cpython/blob/3.8/Lib/typing.py#L1943) that IMO shouldn't be ignored. Additionally, I'm hesitant to change the base class of `addbase` since it's special-cased in this file and elsewhere.

Reply:
But bytearray is always compatible with bytes in argument types, so I don't think that override makes a difference (https://docs.python.org/3/library/typing.html#typing.ByteString). Mypy is OK with this file:
```
from typing import IO

def f(x: IO[bytes]) -> None:
    x.write(bytearray(b''))
```
Feel free to leave the base class alone though. I'm more concerned about usages in argument positions.

Reply:
Alright, that makes sense. Done.